### PR TITLE
[# 101523656] In development, uploading case resources fails for certain versions of mysql

### PR DESCRIPTION
**Pull request by ghost**
Tracker #101523656

part 2 of 2
https://github.com/ConsultingMD/ConsultingMD/pull/4610

### CODE REVIEW

**Comment by ghost** [comment]
@kenberland @cobbr2 please review. 2 part PR

File: lib/carrierwave/securefile/uploader.rb [hunk 1]
```diff
@@ -10,9 +10,7 @@ def self.secure_file(model=nil, file)
           if configuration.encryption_type.downcase.to_sym == :aes
             aes_key = configuration.aes_key
             if model.respond_to? :aes_key and ( model.aes_key.nil? || model.aes_key == '' )
```
Comment (kenberland) [inline lib/carrierwave/securefile/uploader.rb:12]: Is this just `if model.try(:aes_key)`

so 

``` ruby
model.aes_key = aes_key = SecureRandom.random_bytes(32) if model.try(:aes_key)
```

**Comment by kenberland** [comment]
I'm okay to merge this if you don't care to make my change.

**Comment by ghost** [comment]
Let's merge

**Pull request merged**